app/services/address_validation_service.rb

```diff
@@ -18,4 +18,11 @@ def invalid?
     !@validation.verifications.delivery.success ||
       @validation.street1.include?('PO BOX')
   end
+
+  def error
+    return nil unless @validation.present? &&
+                      @validation.verifications.delivery.errors.present?
+    return 'PO BOX' if @validation.street1.include?('PO BOX')
```

Comment:
Yeah, I thought the acceptance for the story was to give different error messages for both, so I set up this error message. However I realized it was really just saying "make sure we check for PO Box and also update the text of the error message." So I figure just leave in this functionality in case we ever do want to use it to give more descriptive errors.